slf4j/src/test/java/feign/slf4j/Slf4jLoggerTest.java

```diff
@@ -101,9 +101,9 @@ public void logRequestsAndResponses() throws Exception {
     slf4j.logLevel("debug");
     slf4j.expectMessages("DEBUG feign.Logger - [someMethod] A message with 2 formatting tokens."
         + System.lineSeparator() +
-        "DEBUG feign.Logger - [someMethod] ---> GET http://api.example.com HTTP/1.1"
+        "DEBUG feign.Logger - [someMethod] ---> GET http://api.example.com HTTP"
         + System.lineSeparator() +
-        "DEBUG feign.Logger - [someMethod] <--- HTTP/1.1 200 OK (273ms)"
+        "DEBUG feign.Logger - [someMethod] <--- HTTP/unknown 200 OK (273ms)"
```

Comment:
where did this unknown came from?  Can we make previous messages?

Reply:
done